src/deluge/gui/views/session_view.cpp

```diff
@@ -94,6 +94,7 @@ SessionView sessionView{};
 
 SessionView::SessionView() {
 	xScrollBeforeFollowingAutoExtendingLinearRecording = -1;
+	createClip = false;
```

Comment:
what is this used for?

Reply:
It's used after the clip creation yield to know if you should proceed with clip creation or not - eg if you cancelled out then it doesn't create a clip

Reply:
I meant the line - it's already initialized to false (that's what createClip{false} does). Is there a case where that's not working?

Reply:
> I meant the line - it's already initialized to false (that's what createClip{false} does). Is there a case where that's not working?

ah yes not necessary, just an oversight i'll remove it